scripts/ci/gitlab/pipeline/build.yml

```diff
@@ -21,7 +21,7 @@ build-linux-stable:
     # Ensure we run the UI tests.
     RUN_UI_TESTS: 1
   script:
-    - time cargo build --locked --profile testnet --features pyroscope --verbose --bin polkadot
+    - time cargo build --locked --profile testnet --features pyroscope,fast-runtime --verbose --bin polkadot
```

Comment:
@PierreBesson do you see any problem with changing our polkadot-debug docker images with `fast-runtime` feature?
cc @pepoviola